app/components/articleSearch/components/searchItem/authors.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../../../variables";
+
+.authors {
+  display: inline;
+
+  .authorHIndex {
+    display: inline-block;
+    height: 18px;
+    border-radius: 100px;
```

Comment:
If you want to describe circle, use 50% instead of 100px.
The absolute value depends on the size of the parent object.

Reply:
Design doesn't want to make circle but ellipse so I wrote like that